lib/dqt.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Dqt
+  def self.configuration
+    @configuration ||= Configuration.new
+  end
+
+  def self.configure
+    yield(configuration) if block_given?
+  end
```

Comment:
@blackrat does this look like something that we can use `InitializeWithConfig` for? I am just going off of name similarity, I haven't checked the capabilities or use cases :P